app/javascript/components/subject/MonitoringStatus.js

```diff
@@ -41,8 +41,8 @@ class MonitoringStatus extends React.Component {
       isolation_status: props.patient.isolation ? 'Isolation' : 'Exposure',
       pause_notifications: props.patient.pause_notifications,
       loading: false,
-      apply_to_group_cm_only: false,
```

Comment:
I also see these variables in `app/javascript/components/subject/LastDateExposure.js` - does that need a refactor as well?

Reply:
no - we want this to get into this conditional: `elsif params.permit(:apply_to_group_cm_only)[:apply_to_group_cm_only]`